recirq/quantum_chess/quantum_board.py

```diff
@@ -93,15 +97,14 @@ def __init__(
         self.error_mitigation = error_mitigation
         self.noise_mitigation = noise_mitigation
 
-        # None if there is no cache, stores the repetition number if there is a cache.
-        self.accumulations_repetitions = None
-        self.board_accumulations_repetitions = None
+        # Stores the repetition number if there is a cache.
+        self.board_accumulations_repetitions = -1
```

Comment:
Why are we using -1 as a magic number to mean no repetitions rather than None?  I don't think this is a good idea.  Even if you persist in doing it, we should name it as a constant value.

Reply:
Because it is an int which is smaller than any possible number of repetitions. So it gives the right result when comparing the cached repetitions to the requested number of repetitions

Reply:
I would still make this a constant, such as UNINITIALIZED_CACHE_VALUE or something like that.

Reply:
Done